test/c/derive.c

```diff
@@ -192,6 +192,78 @@ void derive_single_line_with_custom_knots(CuTest *tc)
 	TS_END_TRY
 }
 
+void derive_single_line_with_custom_knots_degree_2(CuTest *tc)
```

Comment:
A spline of degree two is a parabola. So the name should rather be `derive_single_parabola_with_custom_knots`

Reply:
Will fix. I was trying to highlight that this test is different than the previous because with `degree == 1`, the scaling is now invoked as per [L1479](https://github.com/msteinbeck/tinyspline/blob/8b809ac026d542933723221f6124d15f0d4a44f5/src/tinyspline.c#L1479) of `tinyspline.c`. I'll add comments to this end and change the name as suggested.

Reply:
Renamed in 39b52d578956fad00b6190b7dcee5cc6e971416b and comments added in 6a1db70f7f646a4f5196b109556685f646490267